### DISCUSSION TITLE
Can I rename my chats , snippets or websites in pieces ?

### DISCUSSION THREAD

**Discussion by Codeeyyy** [pieces-app/support#381] [Q&A]
Can I rename my chats , snippets or websites in pieces ? I am so frustrated that I am not able to rename chats , snippets etc is there any possible way to do so?

**Comment by mack-at-pieces** [comment]
Hi @Codeeyyy You are absolutely able to rename your snippets & chats. How you do so would depend on which Pieces product/plugin you're attempting to do so. If you have a spare 15 minutes sometime this week, I'd be happy to show you how to do so and answer any other questions you have. You can book a time on my calendar here: https://calendar.app.google/k5zaUHuHFPnmbmyM6
